dts/bindings/radio_coex/nordic,bluetooth-radio-coex-three-wire.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of an external radio coexistence setup that has a
+    three-pin control interface (REQUEST, GRANT, PRIORITY/DIRECTION). This
+    extends that interface by adding additional parameters needed by the
+    Softdevice Controller's implementation.
+
+compatible: "nordic,bluetooth-radio-coex-three-wire"
```

Comment:
Sorry for an incoming influx of basic questions, but I don't have any experience in this area.

Is this really a piece of hardware for which we are the vendor? Can you refer me to a datasheet if so?

If not, this is the wrong thing to do; see: https://docs.zephyrproject.org/latest/guides/dts/bindings.html#vendor-prefixes

Reply:
this has now been renamed.